linetools/lists/linelist.py

```diff
@@ -117,9 +118,9 @@ def name(self):
 
     @property
     def wrest(self):
-        """ Return the rest wavelengths
+        """ Return the rest wavelengths as a Quantity array
         """
-        return self._data['wrest']
+        return Quantity(self._data['wrest'])
```

Comment:
The Table does have units.  And by using 
Quantity here, they get 'ingested'.  Rather nice, eh?